app/assets/javascripts/price_policy.js

```diff
@@ -72,6 +72,22 @@ $(document).ready(function() {
     }
   });
 
+  $(".js--baseRate").on("change", function (event) {
+    setRate(event.target)
+  });
+
+  function setRate(params) {
+    const columnIndex = params.dataset.index;
+    const stepBaseRate = params.value;
+    $(`input[name*='duration_rates_attributes][${columnIndex}][rate]']`).val(
+      stepBaseRate
+    );
+  }
+
+  $(".js--baseRate").each(function (_index, element) {
```

Comment:
Small nitpick, but I'd move this up to line 75.  First we set up the page on load, then register the "change" event listener, then define the function.